src/components/EmailList/index.jsx

```diff
@@ -0,0 +1,125 @@
+import EmailItem from "../EmailItem";
+import { useEffect, useState } from "react"
+import EmailView from "../EmailView";
+
+function EmailList() {
+
+    const [emailArray, setEmailArray] = useState(false);
+    const [emailID, setEmailID] = useState(0);
+    const [emailRead, setEmailRead] = useState(0);
+    const [refreshJson, setRefreshJson] = useState(false);
```

Comment:
```suggestion
    const [refreshEmailList, setRefreshEmailList] = useState(false);
```
```suggestion
    const [emailListNeedsRefresh, setEmailListNeedsRefresh] = useState(false);
```

Call it what it does. `refreshJson` could mean *anything*.

Hint: You can easily change the names of things in VSCode everywhere at once by right-clicking on them and selecting 'Rename Symbol'.